openssl/ca.cnf

```diff
@@ -0,0 +1,243 @@
+# Include upstream ca & req apps configuration
+# Changes from upstream:
+# - default_days increase to 825 - maximum allowed on Mac OS X
+# - default_bits increase to 3072 for NIST SP 800-131A Rev. 3
+# - countryName_default set to US (Chainguard Inc)
+# - sateOrProvinceName_default set to Washington (Chainguard Inc)
+# - organizationName_default set to Chainguard Images
+#
+# Configuration for TSA & CMP apps is not provided for now
+#
+# These defaults are more feature-proof, and provide better
+# tracebility, and prevent accidental privacy leaks to the CMP demo
+# server.
+
+# Comment out the next line to ignore configuration errors
+config_diagnostics = 1
+
+[ ca ]
+default_ca	= CA_default		# The default ca section
+
+[ CA_default ]
+
+dir		= ./demoCA		# Where everything is kept
+certs		= $dir/certs		# Where the issued certs are kept
+crl_dir		= $dir/crl		# Where the issued crl are kept
+database	= $dir/index.txt	# database index file.
+#unique_subject	= no			# Set to 'no' to allow creation of
+					# several certs with same subject.
+new_certs_dir	= $dir/newcerts		# default place for new certs.
+
+certificate	= $dir/cacert.pem 	# The CA certificate
+serial		= $dir/serial 		# The current serial number
+crlnumber	= $dir/crlnumber	# the current crl number
+					# must be commented out to leave a V1 CRL
+crl		= $dir/crl.pem 		# The current CRL
+private_key	= $dir/private/cakey.pem # The private key
+
+x509_extensions	= usr_cert		# The extensions to add to the cert
+
+# Comment out the following two lines for the "traditional"
+# (and highly broken) format.
+name_opt 	= ca_default		# Subject Name options
+cert_opt 	= ca_default		# Certificate field options
+
+# Extension copying option: use with caution.
+# copy_extensions = copy
+
+# Extensions to add to a CRL. Note: Netscape communicator chokes on V2 CRLs
+# so this is commented out by default to leave a V1 CRL.
```

Comment:
Yes, it did made me double-take this when I read it line-by-line too.

I think fax maybe off, but I will make sure to try to do a pager message to them.

Also it's over 20 years now since netscape died.